plugins/woocommerce/src/Database/Migrations/CustomOrderTable/CLIRunner.php

```diff
@@ -1213,6 +1226,104 @@ public function backfill( array $args = array(), array $assoc_args = array() ) {
 		);
 	}
 
+	/**
+	 * Show the list of WooCommerce-aware plugins known to be compatible, incompatible or without compatibility declaration for HPOS. Note that inactive plugins will always be listed in the "uncertain" list.
+	 *
+	 * [--include-inactive]
+	 * : Include inactive plugins in the list.
+	 *
+	 * [--display-filenames]
+	 * : Print plugin file names instead of plugin names.
+	 *
+	 * @since 9.1.0
+	 *
+	 * @param array $args       Positional arguments passed to the command.
+	 * @param array $assoc_args Associative arguments (options) passed to the command.
+	 */
+	public function compatibility_info( array $args = array(), array $assoc_args = array() ) {
```

Comment:
```suggestion
	public function compatibility_info( array $args = array(), array $assoc_args = array() ): void {
```

(Optional) since it's a shiny brand new method, we could include a return type.